pom.xml

```diff
@@ -1728,6 +1729,7 @@
     <restrict-imports.enforcer.version>0.14.0</restrict-imports.enforcer.version>
     <!-- Location of test resources -->
     <test.build.classes>${project.build.directory}/test-classes</test.build.classes>
+    <test.tmp.dir>${project.build.directory}</test.tmp.dir>
```

Comment:
Ok so we will start writing to project bin directory. Pardon my ignorance but if build process gets killed, this directory for build logs will be cleaned right? (because it's containerized)

Reply:
This is just for tmp files, which should be cleaned after the builds. We will write build logs to other places.

Reply:
Got it, Thanks

Reply:
And this should go to all active branches to fix nightlies right?

Reply:
Yes.